### PR TITLE
refactor: update plugnet-dev-travis-build

### PR DESCRIPTION
**Pull request by AlexWZ0129**

### CODE REVIEW

File: packages/dev/scripts/plugnet-dev-travis-build.sh [hunk 1]
```diff
@@ -110,162 +110,162 @@ function npm_bump () {
   echo "*** Npm increment completed"
 }
 
-function npm_get_version () {
```
Comment (ianhe8x) [inline packages/dev/scripts/plugnet-dev-travis-build.sh]: keep function definitions but don't call them

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
+branches:
+  only:
+  - master
 deploy:
   skip_cleanup: true
   provider: script
   script: ./packages/dev/scripts/plugnet-dev-travis-deploy.sh
   on:
     branch: master
+    tags: true
```
Comment (ianhe8x) [inline .travis.yml]: do you need to define a tag filter?

Comment (AlexWZ0129) [reply .travis.yml]: let me check

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
+branches:
+  only:
+  - master
 deploy:
   skip_cleanup: true
   provider: script
   script: ./packages/dev/scripts/plugnet-dev-travis-deploy.sh
   on:
-    branch: master
+    tags: true
```
Comment (ianhe8x) [inline .travis.yml]: disable deploy scripts until completely tested

**Pull request merged**